packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -80,17 +88,29 @@ class ButtonGroupButton extends React.Component {
     // The onFocus event doesn't get triggered in some browsers, hence, the focus state needs to be managed here.
     if (event.nativeEvent.keyCode === KeyCode.KEY_TAB) {
       this.setState({ focused: true });
+      this.shouldShowFocus = true;
     }
 
     if (this.props.onKeyUp) {
       this.props.onKeyUp(event);
     }
   }
 
+  handleFocus(event) {
+    if (this.shouldShowFocus) {
+      this.setState({ focused: true });
+    }
+
+    if (this.props.onFocus) {
+      this.props.onFocus(event);
+    }
+  }
+
   render() {
     const {
       icon,
       isDisabled,
+      onFocus,
```

Comment:
We should add this to the propTypes & add a description

Reply:
It is already present at [line 30](https://github.com/cerner/terra-core/blob/master/packages/terra-button-group/src/ButtonGroupButton.jsx#L30).

Reply:
oh 😄 easy update then 😋